lego/apps/events/serializers/events.py

```diff
@@ -127,6 +127,7 @@ class Meta:
             "registration_count",
             "survey",
             "use_consent",
+            "youtube_url",
```

Comment:
Same as articles, what happens on the front page or event-list when the backend doesn't pass a youtube url for the header?

Reply:
Realized I might have misunderstood - is the idea to upload a header for frontpage/list use, and have the youtube video for the detail view?

Reply:
Yep, the idea was that the cover photo should be shown for lists and the youtube video will be shown on detail view 👍